be/src/common/status.h

```diff
@@ -399,6 +409,9 @@ class STATUS_ATTRIBUTE Status {
     //    _state[5.. 5 + len1]                == message
     //    _state[5 + len1 .. 5 + len1 + len2] == context
     const char* _state = nullptr;
+
+    // message to user clients that does not contain sensitive content
+    const char* _state_for_client = nullptr;
```

Comment:
If you add a new member here, this Status structure will be doubled.
This will influence almost all the functions in StarRocks.
Could you please why this member was introduced and is there any other method to achieve it?

Reply:
I'd prefer add a new function to remove context info from the Status, or create a new Status from another withou context info.